docs/wiki/Solution creation.md

```diff
@@ -0,0 +1,302 @@
+This section shows you how you can orchestrate a deployment using multiple resource modules.
```

Comment:
What should we do with the module usage section?

Reply:
I guess either remove it, or, reduce it to 'only' the part aside from the orchestration. But it could link to the `Solution creation.md`. What do you think?

Reply:
I'd remove the part which is covered by the solution creation page and link that page.

Reply:
done